docs/root/start/_include/dockerhub-images.csv

```diff
@@ -0,0 +1,9 @@
+,,stable,stable,master,master
+Docker image,description,amd64,arm64,amd64,arm64
+`envoyproxy/envoy <https://hub.docker.com/r/envoyproxy/envoy/tags/>`_,Release binary with symbols stripped on top of an Ubuntu Bionic base.,v1.16-latest,v1.16-latest,,
+`envoyproxy/envoy-alpine <https://hub.docker.com/r/envoyproxy/envoy-alpine/tags/>`_,Release binary with symbols stripped on top of a **glibc** alpine base.,v1.16-latest,,,
+`envoyproxy/envoy-debug <https://hub.docker.com/r/envoyproxy/envoy-debug/tags/>`_,Release binary with debug symbols on top of an Ubuntu Bionic base.,v1.16-latest,v1.16-latest,,
+`envoyproxy/envoy-dev <https://hub.docker.com/r/envoyproxy/envoy-dev/tags/>`_,Release binary with symbols stripped on top of an Ubuntu Bionic base.,,,latest,latest
+`envoyproxy/envoy-alpine-dev <https://hub.docker.com/r/envoyproxy/envoy-alpine-dev/tags/>`_,Release binary with symbols stripped on top of a **glibc** alpine base.,,,latest,
+`envoyproxy/envoy-debug-dev <https://hub.docker.com/r/envoyproxy/envoy-debug-dev/tags/>`_,Release binary with debug symbols on top of an Ubuntu Bionic base.,,,latest,latest
+`envoyproxy/envoy-build-ubuntu <https://hub.docker.com/r/envoyproxy/envoy-debug-dev/tags/>`_,Build image which includes tools for building multi-arch Envoy and containers.,,,latest(?),latest(?)
```

Comment:
not sure what to do with `latest(?)`

Reply:
So per the other issue, I'm fine with adding latest for this image. @lizan?

Stepping back though, WDYT about moving the documentation of this container to the building section? IMO it doesn't really fit in this section?

Reply:
> Stepping back though, WDYT about moving the documentation of this container to the building section? IMO it doesn't really fit in this section?

not sure if i agree entirely. The existing docs spread info about images in a way that was really hard to grok. Ive tried to put info about all active images in one place - to be authoritative.

i would have separated the table with subheadings, but after quite a bit of trying i wasnt able to get sphinx to do colspans (it doesnt seem to be documented that it can, but it is part of the rst spec afaict)

ultimately moving it doesnt resolve the `latest(?)` issue it just kicks it to another page

Reply:
> Ive tried to put info about all active images in one place - to be authoritative.

that said, i also think if we do keep it here we need to make clear it doesnt contain  an envoy binary, is very large etc - perhaps putting in a separate table.

Reply:
In terms of what to do with the release versions and the SHA if the build image, I think this can very easily be grammatically generated as part of `conf.py` in the sphinx build. The version can be inferred from: https://github.com/envoyproxy/envoy/blob/master/VERSION (you can even have the table generated different depending on whether it's release docs or not, up to you). The build SHA can be pulled from: https://github.com/envoyproxy/envoy/blob/master/ci/envoy_build_sha.sh.

My advice is to just get something working and we can bikeshed about the "optimal" flow later.

Reply:
on the basis that generating this csv file is outside the scope of this PR i have put "see dockerhub" here